src/fal/cli.py

```diff
@@ -150,6 +185,14 @@ def _run(
         )
         project = FalProject(faldbt)
         models = project.get_filtered_models(all, selector_flags, before)
+
+        # TODO: remove `action="extend"` to match exactly what dbt does
+        if selects_count > 1:
+            dbt.exceptions.warn_or_error(
+                "Passing multiple --select/--model flags to fal is deprecatd. Please use model selection like dbt.",
```

Comment:
can you add an example like:

`fal run -s a b` instead of `fal run -s a -s b`

Reply:
adding it in a new pr/commit